recipes-tag/h5py/run_test.py

```diff
@@ -0,0 +1,24 @@
+import h5py
+import h5py._conv
+import h5py._errors
+import h5py._objects
+import h5py._proxy
+import h5py.defs
+import h5py.h5
+import h5py.h5a
+import h5py.h5d
+import h5py.h5f
+import h5py.h5fd
+import h5py.h5g
+import h5py.h5i
+import h5py.h5l
+import h5py.h5o
+import h5py.h5p
+import h5py.h5r
+import h5py.h5s
+import h5py.h5t
+import h5py.h5z
+import h5py.utils
```

Comment:
I installed latest version 2.7 of h5py from conda-forge, and tried this command,
```
In [3]: h5py.h5.HDF5_VERSION_COMPILED_AGAINST
   ...:
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-34c104899b6f> in <module>()
----> 1 h5py.h5.HDF5_VERSION_COMPILED_AGAINST

AttributeError: module 'h5py.h5' has no attribute 'HDF5_VERSION_COMPILED_AGAINST'
```